flow/alerting/alerting.go

```diff
@@ -439,9 +444,14 @@ func (a *Alerter) LogFlowError(ctx context.Context, flowName string, err error)
 		logger.Warn("failed to insert flow error", slog.Any("error", err))
 		return
 	}
+
+	errorClassString := ""
+
 	var tags []string
 	if errors.Is(err, context.Canceled) {
 		tags = append(tags, string(shared.ErrTypeCanceled))
+		// TODO this is only set for context.Canceled, other types need to be added too
```

Comment:
TODO

Reply:
thinking of a couple of things, will take up later